### PR TITLE
Add Toast

### PR DESCRIPTION
**Pull request by Citrullin**
Usage:
`pushToast({languageId: 'error'})`

### CODE REVIEW

File: src/components/Toast.svelte [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+    import {toastStore} from "../stores/stores";
+    import { slide } from 'svelte/transition';
```
Comment (mehlium) [inline src/components/Toast.svelte:3]: Nice! :trophy:

File: src/components/Toast.svelte [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            setTimeout(() => {
+                visible = false;
+            }, timeout)
+        }
+    })
+</script>
+
+{#if visible}
+    <div class="kui-toast" out:slide>
+        <p class="kui-pri">{body}</p>
```
Comment (mehlium) [inline src/components/Toast.svelte]: I think most (if not all) toasts will be some languageId, so we could use `data-l10n-id` here.

**Review by mehlium — APPROVED** [review]
Good job! Slide worked really well with this!

File: src/components/Toast.svelte (hunk 2 above)
Comment (Citrullin) [reply src/components/Toast.svelte]: Ohh yeah, fine for me :)

**Pull request merged**